src/views/empty-state.html

```diff
@@ -9,7 +9,7 @@ <h4 id="blank-state-pf-title-{{$id}}" class="h1 blank-state-pf-title">
     {{$ctrl.config.info}}
   </p>
   <p id="blank-state-pf-helpLink-{{$id}}" class="blank-state-pf-helpLink" ng-if="$ctrl.config.helpLink" ng-click="$ctrl.config.helpLink.urlAction()">
-    {{$ctrl.config.helpLink.label}} <a href="{{$ctrl.config.helpLink.url}}">{{$ctrl.config.helpLink.urlLabel}}</a>.
+    {{$ctrl.config.helpLink.label}} <a class="pf-helplink" ng-attr-href="{{$ctrl.config.helpLink.url ? $ctrl.config.helpLink.url : undefined}}">{{$ctrl.config.helpLink.urlLabel}}</a>.
```

Comment:
this fails accessibility via keyboard navigation

Reply:
Any reason not to use...

```html
<a ng-if="$ctrl.config.helpLink.url" class="pf-helplink" ng-href="{{$ctrl.config.helpLink.url}}">{{$ctrl.config.helpLink.urlLabel}}</a>
<a ng-if="!$ctrl.config.helpLink.url" class="pf-helplink" href="" role="button">{{$ctrl.config.helpLink.urlLabel}}</a>
```

Reply:
Did something similar but with a href and a button.

Reply:
> Did something similar but with a href and a button.

even better :) thanks